test/models/patient_test.rb

```diff
@@ -524,6 +524,13 @@ class PatientTest < ActiveSupport::TestCase
 
       # it 'should update to "Pledge Paid" after a pledge has been paid' do
       # end
+      it 'should update to "No contact in 120 days" after 120ish days of no calls' do
+        create :call, patient: @patient, status: 'Reached patient', created_at: 121.days.ago
+        assert_equal Patient::STATUSES[:dropoff], @patient.status
+
+        create :call, patient: @patient, status: 'Left voicemail', created_at: 119.days.ago
```

Comment:
The twitchy bit in my head wants this to be 120 instead of 119 to make the boundary clear, but that isn't critical.